samples/graph-appcatalog-lifecycle/csharp/AppCatalogSample/Helper/AppCatalog.cs

```diff
@@ -0,0 +1,402 @@
+﻿using AdaptiveCards;
+using Microsoft.Bot.Builder;
+using Microsoft.Bot.Schema;
+using Microsoft.Graph;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace AppCatalogSample.Helper
+{
+    public class AppCatalogHelper
+    {
+        public static string gToken { get; set; }
```

Comment:
Access token can not be static field, this will overlap if multiple users are using this app. Could you please change this to instance filed?